### PR TITLE
Cleanups galore

### PR DESCRIPTION
**Pull request by adriancole**
* Changes groupId to match jitpack convention until we settle things
* Changes license and formatting to Zipkin
* Replaces pom and travis details from normal Zipkin (minus publish config)
* Strips junit legacy annotations
* Makes things package private where possible
* Removes dependency on zipkin-reporter
* enforces JDK 11
* Other fuzz I can't recall :P

### CODE REVIEW

**Comment by adriancole** [comment]
Note: on my workstation, docker flips out like this on a couple tests eventhough it seems travis doesn't..

```

java.util.concurrent.CompletionException: java.net.UnknownHostException: failed to resolve 'macbook-pro-7.local' after 3 queries 

	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:331)
```

**Comment by adriancole** [comment]
testing jitpack build https://jitpack.io/#openzipkin-contrib/zipkin-storage-kafka

**Comment by adriancole** [comment]
jitpack works now I think

```bash
$ TAG=cleanups-SNAPSHOT
$ curl -sSL https://jitpack.io/com/github/openzipkin-contrib/zipkin-storage-kafka/zipkin-autoconfigure-storage-kafka/${TAG}/zipkin-autoconfigure-storage-kafka-${TAG}-module.jar > storage-kafka.jar
```

**Comment by adriancole** [comment]
@jeqo I think this is ready to merge. Once merged we can start integrating more. cheers!

**Comment by jeqo** [comment]
Only Docker related files need to be fixed but we can do it in another PR.

File: storage/src/test/java/zipkin2/storage/kafka/KafkaStorageIT.java [hunk 1]
```diff
@@ -49,21 +51,23 @@
...
 
-  @Container private KafkaContainer kafka = new KafkaContainer("5.3.0");
+  static final long TODAY = System.currentTimeMillis();
 
-  private Duration traceTimeout;
-  private KafkaStorage storage;
-  private Properties consumerConfig;
-  private KafkaProducer<String, List<Span>> tracesProducer;
-  private KafkaProducer<String, DependencyLink> dependencyProducer;
+  // TODO: does this need to be confluent container?
```
Comment (jeqo) [inline storage/src/test/java/zipkin2/storage/kafka/KafkaStorageIT.java]: @adriancole nope, it is just the one used by testcontainers https://www.testcontainers.org/modules/kafka/

we could use our own via generic containers tho, wdyt?

Comment (adriancole) [reply storage/src/test/java/zipkin2/storage/kafka/KafkaStorageIT.java]: yeah if we use ours it will reduce questions I think at least in our own group

**Pull request merged**